cmd/root.go

```diff
@@ -329,9 +340,26 @@ func initKonnectConfig() error {
 		password = strings.TrimRight(password, "\n")
 	}
 
+	token := viper.GetString("konnect-token")
+	tokenFile := viper.GetString("konnect-token-file")
+	// read from token file only if token is not supplied using an
+	// environment variable or flag
+	if token == "" && tokenFile != "" {
```

Comment:
Would it be better to mark the flags as mutually exclusive instead?

Reply:
I added a mutually exclusive check, but it seems having these as env vars or coming from a deck config file would by-pass the check, so I think we still need something like this...so we either add more checks to enforce mutually exclusivity, or we keep this code which is also consistent with what we were already doing in regards to `konnect-password` and `konnect-password-file`

Reply:
I see. Your call to mark these flags is exclusive or not since it is double work.

Reply:
For now I removed the cobra checks and kept the existing logic which is consistent with other parts of the code.